src/sentry/mail/adapter.py

```diff
@@ -60,11 +60,20 @@ def rule_notify(
                 "The default behavior for notification de-duplication does not support args"
             )
 
+        skip_digests = False
+        # Check if this is notification test
+        if len(rules) == 1 and rules[0].id is None:
+            skip_digests = True
+
```

Comment:
I don't think this is the best, but the only other way I see is to add an extra arg to every single function that `after` returns.

Reply:
I'd prefer to isolate this to the action. We could make `skip_digests` a parameter here, then here we could pass it based on some criteria: https://github.com/getsentry/sentry/blob/33370eea81f15f659ea0f18a7c379e269f480935/src/sentry/mail/actions.py#L21-L38

We could potentially even inject it into the `data` for this action, then pass `self.data.get("skip_digests", False)` to `rule_notify`